<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>491</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{border:none;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_4{
	border:none;
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>51</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#listWidget{
	border:none;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>359</y>
     <width>491</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_3{
	background-image: none;
	background-color: rgb(255, 255, 255);
	border-top: 1px solid gray;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>41</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource.qrc">
       <normaloff>:/images/1.png</normaloff>:/images/1.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>41</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resource.qrc">
       <normaloff>:/images/icons8-文件-100.png</normaloff>:/images/icons8-文件-100.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="content">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
